opentelemetry-exporters-newrelic/build.gradle.kts

```diff
@@ -3,13 +3,34 @@ dependencies {
     val openTelemetryVersion = "0.6.0"
 
     api("com.newrelic.telemetry:telemetry:$newRelicTelemetrySdkVersion")
+    implementation("org.slf4j:slf4j-simple:1.7.26")
     implementation("com.newrelic.telemetry:telemetry-http-okhttp:$newRelicTelemetrySdkVersion")
     implementation("io.opentelemetry:opentelemetry-sdk:$openTelemetryVersion")
 
-    testRuntimeOnly("org.slf4j:slf4j-simple:1.7.26")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.4.2")
     testImplementation("org.junit.jupiter:junit-jupiter-api:5.4.2")
     testImplementation("org.mockito:mockito-core:3.0.0")
     testImplementation("org.mockito:mockito-junit-jupiter:3.0.0")
     testImplementation("com.google.guava:guava:28.0-jre")
 }
+
+tasks {
+    val propertiesDir = "build/generated/properties"
+    val versionFilename = "newrelic.exporter.version"
+    sourceSets.get("main").output.dir(mapOf("builtBy" to "generateVersionResource"), propertiesDir)
```

Comment:
what is this `"builtBy"` thing? I've never seen that before.

Reply:
I guess it declares what thing creates it?